orchestration/src/test/java/com/sap/ai/sdk/orchestration/client/OrchestrationUnitTest.java

```diff
@@ -174,54 +171,54 @@ void template() throws IOException {
 
     // verify that null fields are absent from the sent request
     final String request =
-        new String(
-            getClass()
-                .getClassLoader()
-                .getResourceAsStream("templatingRequest.json")
-                .readAllBytes());
+            new String(
+                    getClass()
+                            .getClassLoader()
+                            .getResourceAsStream("templatingRequest.json")
+                            .readAllBytes());
```

Comment:
Please install the `google-java-format` plugin and reformat.
Also, should we enable spotless on generated files in this PR or in a next PR?

Reply:
I ran format on the test class and pom file.

I don't understand why you want the files to me formatted when it was you who disabled the plugin in the first place.